llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -73,7 +73,7 @@ def SVEUnsupported : AArch64Unsupported {
                       SVE2Unsupported.F);
 }
 
-let F = [HasSME2p1, HasSVE2p1_or_HasSME2p1] in
+let F = [HasSME2p1, HasSVE2p1_or_HasSME2p1, HasSVE2p1orSSVE_AES] in
```

Comment:
The existing label `HasSVE2p1_or_HasSME2p1` is easier to read with underscores. To be consistent, could this be `HasSVE2p1_or_HasSSVE_AES`?

Reply:
Sure, let's tidy up after.